src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1810,10 +1810,10 @@ limitations under the License.
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
+            <include cluster="Binding" client="false" server="true" clientLocked="true" serverLocked="true">
```

Comment:
Binding client is enabled due to the mandatory time sync client, right?

Reply:
Yes